test/src/e2e_vm_tests/test_programs/should_pass/return_in_strange_positions/src/main.sw

```diff
@@ -0,0 +1,97 @@
+script;
+
+// This file tests the reported errors and warnings in various instances when a 'return'
+// occurs in a non-statement position. This is allowed, but will often result in
+// unreachable code or similar warning situations.
+
+pub struct S { x : u64, y : u64, }
+pub enum Enum {
+    A: u64,
```

Comment:
```suggestion
    A: (u64, u64),
```

Reply:
Just to stress the CFG more like in other cases where we've used `return 42` and `return 43` wherever we could. Sorry for realizing this only know. When I passed the suggestion for the first time this suggestion didn't cross my mind. And also considering the latest appearance of ICEs I think stressing the compiler and the CFG more makes sense.